templates/nginx/nginx.conf.j2

```diff
@@ -28,9 +28,18 @@ http {
     keepalive_timeout {{ nginx_keepalive_timeout }};
     keepalive_requests {{ nginx_keepalive_requests }};
     reset_timedout_connection on;
-
-    proxy_buffering on;
-    proxy_buffers 8 4k;
+{% if nginx_proxy_buffering  is defined %}
```

Comment:
Aren't this variables always defined in `defaults/main.yml`?

Reply:
Yes, in the beginning these variables weren't going to be defined in defaults, it was a late decision.

Reply:
then, this checks can be removed from the template? :-)